colossalai/pipeline/schedule/zero_bubble_pp.py

```diff
@@ -220,35 +226,47 @@ def recv_forward(self, model_chunk_id: int, prev_rank: int = None) -> Tuple[Any,
                 # do nothing; cause u are chunk 0 in first rank, u have no prev rank;
                 #################
                 if self.stage_manager.is_first_stage(ignore_chunk=True):
-                    return None, []
+                    # return None, []
+                    return []
```

Comment:
If you change return type, the signature should also be updated

Reply:
> If you change return type, the signature should also be updated

fixed in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/fa3ccda8ee6da5fb5751ff93b5226d757e4a5e79; Only return List type item now;